tfhe/src/shortint/oprf.rs

```diff
@@ -85,7 +108,11 @@ impl ServerKey {
             "The number of random bits asked for (={random_bits_count}) is bigger than full_bits_count (={full_bits_count})"
         );
 
-        let ct = self.create_random_from_seed(seed);
+        assert_eq!(self.pbs_order, PBSOrder::KeyswitchBootstrap);
```

Comment:
why the assert ? We could keyswitch to get back to the right key size for the PBS_KS case right ?

Reply:
Yes, looks like it would be ok